developerportal/settings/base.py

```diff
@@ -336,3 +314,55 @@
         "OPTIONS": {"CLIENT_CLASS": "django_redis.client.DefaultClient"},
     }
 }
+
+# Mozilla OpenID Connect / Auth0 configuration
+
+# How frequently do we check with the provider that the
+# user still exists and is authorised?
+OIDC_RP_SIGN_ALGO = "RS256"
+
+OIDC_RENEW_ID_TOKEN_EXPIRY_SECONDS = (
+    15 * 60
+)  # 15 mins is the default, but re-expressed here for visibility
+OIDC_CREATE_USER = False  # We don't want stop drive-by signups
+
+OIDC_RP_CLIENT_ID = os.environ.get("OIDC_RP_CLIENT_ID")
```

Comment:
TODO: amend these so that only the secret and ID come from the env -- @limed has confirmed that @jdow from the IAM team is fine with the full OIDC URLs living in the codebase. (This makes sense, too, because they are not secret, given that anyone will know them after clicking the SSO sign in button)